reference/api/settings.mdx

```diff
@@ -152,6 +152,10 @@ If the provided index does not exist, it will be created.
 
 <CodeSamples id="update_settings_1" />
 
+<Capsule intent="warning">
+If one or more settings have an error, Meilisearch will not return any results. In case of multiple errors, the error message will only address the first erroneous setting.
```

Comment:
Where is this change coming from? It does not seem to be related to the issue.

Reply:
It's not. But you get the same behavior here, so I thought we should update this as well

Reply:
Ah, well there are no "results" when updating settings so the first sentence should be changed or removed.